ddtrace/internal/writer.py

```diff
@@ -263,9 +261,11 @@ def __init__(
                 }
             )
 
-        self._encoder = Encoder()
+        self._encoder = Encoder(
```

Comment:
this assumes only our custom encoder can be used, but what about the JSON or JSONv2 encoder?

Reply:
That's right, those would have to be refactored too

Reply:
this change isn't needed, only this writer uses this encoder.